### DISCUSSION TITLE
GeoVista Hall of Fame

### DISCUSSION THREAD

**Discussion by dennissergeev** [bjlittle/geovista#309] [Ideas]
Hey @bjlittle,

I've just got an idea (admittedly at this point a rather selfish one as you'll see) of creating a list/gallery of papers/posters/talks etc. with figures/animations created with `geovista`. This could be a part of the `geovista` docs and/or README. People could then submit PRs and add their work to the list, sort of like [this gallery of interesting jupyter notebooks](https://github.com/jupyter/jupyter/wiki#a-gallery-of-interesting-jupyter-notebooks).

I have the first suggestion: https://egusphere.copernicus.org/preprints/2023/egusphere-2023-647/ (See Fig. 1) :smile:

Cheers,
Denis

**Comment by bjlittle** [comment]
@dennissergeev Congrats on your paper :partying_face: :tada:

Very cool!

I think this is a great idea and our thinking is pretty much aligned. I'd love to see a community grow around `geovista` where people can easily share and showcase how they're using `geovista` for they're workflow, use-case or research.

I'm aiming to boostrap the initial `geovista` documentation in `0.4.0` (see this [comment](https://github.com/bjlittle/geovista/issues/15#issuecomment-1582238616)), so I'm keen for sure to see if we can incorporate your suggestion in this initial step.

How's about I ping you when I get close to that point, and perhaps we can flesh it out and see how it rolls?

**Reply by bjlittle** [reply]
@dennissergeev Also, the significance had finally just dawned on me ...

... https://egusphere.copernicus.org/preprints/2023/egusphere-2023-647/ is the first official citation of `geovista` in an academic paper! :open_mouth: 

![giphy](https://github.com/bjlittle/geovista/assets/2051656/e481c6d1-7447-4393-bd65-f4b11d6567e6)

**Reply by dennissergeev** [reply]
I guess so! First in a long line of papers to come!

**Comment by dennissergeev** [comment]
The paper is finally published! With an extra `geovista`-generated cover image :smile: 

https://doi.org/10.5194/gmd-16-5601-2023

![](https://gmd.copernicus.org/articles/16/5601/2023/gmd-16-5601-2023-avatar-web.png)

**Reply by tkoyama010** [reply]
@dennissergeev Awesome job! Could you also share in [PyVista Discussions](https://github.com/pyvista/pyvista/discussions)?

**Reply by dennissergeev** [reply]
[Done](https://github.com/pyvista/pyvista/discussions/5043#discussion-5721625)! Thanks @tkoyama010

**Reply by bjlittle** [reply]
@dennissergeev Congrats 🍻 

Totally lush viz too! 🤩

**Comment by dennissergeev** [comment]
Another little addition to the GeoVista Hall of Fame :smile:

![lfric_stretched_mesh_dirac_image_competition_2023](https://github.com/bjlittle/geovista/assets/12111288/aac266e0-3ae7-44b6-b3a8-0c224456cebe)

(I made this as an entry for the DiRAC HPC Research Image Competition.)

https://x.com/DiRAC_HPC/status/1743253092764631042?s=20

**Comment by bjlittle** [comment]
@dennissergeev I've opened the doors to the `geovista` [🍬 Candy Store](https://github.com/bjlittle/geovista/discussions/1033)!

A place for the community to openly share their work ... do you want to start the ball rolling? 😉

**Reply by dennissergeev** [reply]
that's brilliant, thanks @bjlittle!
